src/Bridges/Constraint/indicator_sos.jl

```diff
@@ -0,0 +1,185 @@
+            (MOI.ScalarAffineFunction{T}, BC),
+            ]
+end
+
+function MOIB.added_constraint_types(::Type{<:IndicatorSOS1Bridge{T, S}}) where {T, S <: MOI.AbstractScalarSet}
+    return [(MOI.VectorOfVariables, MOI.SOS1{T}),
+            (MOI.ScalarAffineFunction{T}, S),
+           ]
+end
+
+function concrete_bridge_type(::Type{<:IndicatorSOS1Bridge{T}},
+                              ::Type{<:MOI.AbstractVectorFunction},
+                              ::Type{MOI.IndicatorSet{MOI.ACTIVATE_ON_ONE, S}}) where {T, S<:Union{MOI.LessThan, MOI.GreaterThan}}
+    return IndicatorSOS1Bridge{T, S, MOI.ConstraintIndex{MOI.SingleVariable, S}}
+end
+
+function concrete_bridge_type(::Type{<:IndicatorSOS1Bridge{T}},
+                              ::Type{<:MOI.AbstractVectorFunction},
+                              ::Type{MOI.IndicatorSet{MOI.ACTIVATE_ON_ONE, S}}) where {T, S <: MOI.AbstractScalarSet}
+    return IndicatorSOS1Bridge{T, S, Nothing}
+end
+
+# Attributes, Bridge acting as a model
+
+function MOI.get(::IndicatorSOS1Bridge, ::MOI.NumberOfVariables)
+    return 1
+end
+
+function MOI.get(b::IndicatorSOS1Bridge, ::MOI.ListOfVariableIndices)
+    return [b.w_variable_index]
+end
+
+function MOI.get(::IndicatorSOS1Bridge{T, BC, Nothing}, ::MOI.NumberOfConstraints{MOI.SingleVariable, BC}) where {T, BC}
+    return 0
+end
+
+function MOI.get(::IndicatorSOS1Bridge{T, BC, CI}, ::MOI.NumberOfConstraints{MOI.SingleVariable, BC}) where {T, BC, CI <: MOI.ConstraintIndex{MOI.SingleVariable, BC}}
+    return 1
+end
+
+function MOI.get(::IndicatorSOS1Bridge, ::MOI.NumberOfConstraints{MOI.VectorOfVariables, <:MOI.SOS1})
+    return 1
+end
+
+function MOI.get(::IndicatorSOS1Bridge{T, BC}, ::MOI.NumberOfConstraints{MOI.ScalarAffineFunction{T}, BC}) where {T, BC, CI <: MOI.ConstraintIndex{MOI.SingleVariable, BC}}
+    return 1
+end
+
+function MOI.get(b::IndicatorSOS1Bridge{T, BC, CI}, ::MOI.ListOfConstraintIndices{MOI.SingleVariable, BC}) where {T, BC, CI <: MOI.ConstraintIndex}
+    return [b.bound_constraint_index]
+end
+
+function MOI.get(::IndicatorSOS1Bridge{T, BC, Nothing}, ::MOI.ListOfConstraintIndices{MOI.SingleVariable, BC}) where {T, BC}
+    return MOI.ConstraintIndex{MOI.SingleVariable, BC}[]
+end
+
+function MOI.get(b::IndicatorSOS1Bridge{T}, ::MOI.ListOfConstraintIndices{MOI.VectorOfVariables, <:MOI.SOS1}) where {T}
+    return [b.sos_constraint_index]
+end
+
+function MOI.get(b::IndicatorSOS1Bridge{T, BC}, ::MOI.ListOfConstraintIndices{MOI.ScalarAffineFunction{T}, BC}) where {T, BC}
+    return [b.linear_constraint_index]
+end
+
+function MOI.supports(
+    ::MOI.ModelLike,
+    ::MOI.ConstraintPrimalStart,
+    ::Type{<:IndicatorSOS1Bridge})
+    return true
+end
+
+function MOI.get(model::MOI.ModelLike, attr::MOI.ConstraintPrimal, bridge::IndicatorSOS1Bridge)
+    if attr.N == 1
+        return MOI.get(model, MOI.VariablePrimal(), bridge.z_variable_index)
+    end
+    wv = MOI.get(model, MOI.VariablePrimal(), bridge.w_variable_index)
```

Comment:
This should be `MOI.VariablePrimal(attr.N)`

Reply:
how come? w is a scalar variable

Reply:
For the `N`th result, you have `VariablePrimal(N)`, `ConstraintPrimal(N)` and `ConstraintDual(N)`. The value of `ConstraintPrimal(N)` is the value of the function when the variables have values `VariablePrimal(N)`

Reply:
ok, that's not really intuitive, I thought `ConstraintPrimal(N)` would be the primal of the `N`-th function in the vector function. I'll update that